Assets/MixedRealityToolkit/SpatialMapping/Scripts/RemoteMapping/RemoteMappingManager.cs

```diff
@@ -10,7 +9,7 @@
 using UnityEngine.Windows.Speech;
 #endif
 
-namespace MixedRealityToolkit.SpatialMapping.RemoteMapping
+namespace HoloToolkit.Unity.SpatialMapping
```

Comment:
Revert the namespace change for the Dev Branch, or retarget May or June Releases